src/subplots.ts

```diff
@@ -2045,3 +2075,78 @@ export class Histogram extends Frame {
     return [mouse3X, mouse3Y];
   }
 }
+
+
+export class newScatter extends Frame {
+  public points: newPoint2D[] = [];
+  public tooltipAttr: string[];
+  readonly pointsColorFill: string = 'hsl(203, 90%, 85%)';
+  readonly pointsColorStroke: string = 'hsl(0, 0%, 0%)';
+  constructor(
+    public data: any,
+    public width: number,
+    public height: number,
+    public buttons_ON: boolean,
+    public X: number,
+    public Y: number,
+    public canvas_id: string,
+    public is_in_multiplot: boolean = false
+    ) {
+      super(data, width, height, buttons_ON, X, Y, canvas_id, is_in_multiplot);
+      if (!data.tooltip) {this.tooltipAttr = Array.from(this.features.keys()) }
+      else this.tooltipAttr = data.tooltip.attribute;
```

Comment:
Now I know this isn't tooltipAttrition ;)

Reply:
Replaced